app/assets/stylesheets/layout/page_specific/_error_page.scss

```diff
@@ -1,25 +1,77 @@
-.l-error-page {
-  min-height: 600px;
-}
-
 .l-error-page-message {
   @include column(12);
   text-align: center;
-  margin-top: $baseline-unit*10;
   margin-bottom: px(12);
+  background: image-url('error/sorry_261_en.png') center 60px no-repeat;
+  background-size: 261px 189px;
+
+  .theme-cy & {
+    background-image: image-url('error/sorry_261_cy.png');
+  }
 
   & + .directory {
     background-color: transparent;
   }
+
+  @include respond-to($mq-m) {
+    background: image-url('error/sorry_1400_en.jpg') 0 -86px no-repeat;
+    background-size: 100%;
+    padding-top: 42%;
+
+    .theme-cy & {
+      background-image: image-url('error/sorry_1400_cy.jpg');
+      background-position: 0 -60px;
+    }
+  }
+}
+
+.error-page__headline {
+  @extend .visually-hidden;
 }
 
 .error-page__text {
-  @extend .intro;
+  font-family: 'museo_slab';
```

Comment:
I need to show what i've done so far with Irina, and font-weight here will be included - at the moment it will show by default 900 weight (because thats the only one we have included)